### PR TITLE
Update consutil for picocom v1.7

### PR DESCRIPTION
**Pull request by cawand**
**- What I did**
Updated consutil for picocom v1.7 by removing QUIET flag and accounting for /var/lock files
**- How I did it**
Removed any code that used QUIET flag. Added code in `clear()` that removes /var/lock/LCK..ttyUSB<linenumber> file
**- How to verify it**
Run `connect line <linenumber>` or `connect device <devicename>` and `clear line <linenumber>` on device with picocom v1.7. Ensure that you can successfully connect to devices, that you get an error for trying to connect to a busy device, and that `clear` successfully clears a busy line.

-->

### CODE REVIEW

**Comment by cawand** [comment]
Yes - originally I developed consutil using picocom v3, but now we're using picocom v1.7, which doesn't have the -q flag and uses /var/lock files

File: consutil/main.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ def clear(linenum):
         cmd = "sudo kill -SIGTERM " + pid
         click.echo("Sending SIGTERM to process " + pid)
         run_command(cmd)
+        cmd = "sudo rm /var/lock/LCK..ttyUSB" + linenum
```
Comment (jleveque) [inline consutil/main.py]: Suggest to add `-f` flag to `rm` command to help ensure file is deleted.

**Comment by lguohan** [comment]
https://github.com/Azure/sonic-buildimage/pull/1943

**Comment by cawand** [comment]
We can delete this PR since the base image has picocom v3.1

**Pull request closed without merging**